monero-outreach-docs/translations/es/archive/getting_started_es.md

```diff
@@ -0,0 +1,197 @@
+# Empezando a Ayudar a Monero
+
+**_“La comunidad descentralizada de Monero está formada por un grupo de individuos muy dinámico y grupos propagados alrededor del planeta.”_ - 21 de agosto, 2019**
+---
+
+La comunidad descentralizada de Monero está formada por un grupo de individuos muy dinámico y grupos propagados alrededor del planeta. Un hecho poco conocido y emocionante es que Monero tiene la tercera cuenta de colaboradores más alta en todas las criptomonedas. Adicionalmente, hay muchos grupos de trabajo y proyectos que han evolucionado de la mano de Monero y es relativamente fácil empezar a ayudar.
+
+Por favor usen esto como un recurso para aprender sobre Monero, conocer sobre su ecosistema y como PUEDEN involucrarse. Bienvenidos a Monero. Esperamos que se queden.
```

Comment:
_Por favor usen esto como un recurso para aprender sobre Monero, conocer sobre su ecosistema y **cómo** PUEDEN involucrarse._

I would change the "PUEDEN involucrarse" to "PUEDES involucrarte" but is not a grammatical mistake, so it's fine.

Reply:
Changed.